lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -145,7 +145,13 @@ class IRFFITSWriter(Tool):
         --global-alpha-cut 10
         --source-dep
 
-    """
+    To build modified IRFs by specifying a scaling factor applying to the true energy (without using a config file):
+    > lstchain_create_irf_files
+        -g /path/to/DL2_MC_gamma_file.h5
+        -o /path/to/irf.fits.gz
+        --point-like (Only for point_like IRFs)
```

Comment:
if only valid for the point-like case I would make a check in the code, that we are in that case or raise an error otherwise

Reply:
I think it is meant only for a simple example. If so, you may exclude this flag/line